extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/RouteConfig.java

```diff
@@ -27,4 +27,10 @@ public class RouteConfig {
     @ConfigItem
     Map<String, String> annotations;
 
+    /**
+     * Named or numerical port of the Service to bind to
```

Comment:
This is misleading. I think the rest of the code code doesn't handle int values (it always treats it as string)